.github/workflows/build.yml

```diff
@@ -16,19 +16,14 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: [3.6, 3.7, 3.8]
-    
-        # Steps represent a sequence of tasks that will be executed as part of the job
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Can you provide a reason why you moved from the testing multiple versions of python to only one? Is it because of time to run all test in the matrix or incompatibility?

Reply:
I am still a bit conflicted on this. My thoughts are that the multi version testing is better suited for acceptance/end product testing at the moment where having things work with multiple Python versions is more critical. I think the added complexity and more frequent failing builds due to flaky tests outweigh the potential benefit given that these are just unit tests. People developing and running tests on different versions should suffice. I'm open to going back to the matrix build if we feel like it is worth it.

Reply:
In terms of time, the 3.8 build was talking a lot longer to install dependencies for some reason but in theory they should run in parallel and incur low time costs.

Reply:
ACK

Reply:
They will accrue cpu time costs though which at some point will need to be paid for.